src/Iterator/TypedIterator.php

```diff
@@ -28,7 +28,7 @@ final class TypedIterator extends ProxyIterator
 {
     /**
      * @param Iterator<TKey, T> $iterator
-     * @param null|callable(mixed): string $getType
+     * @param callable(T): string $getType
```

Comment:
```suggestion
     * @param null|callable(T): string $getType
```
the callable is nullable here